tests/unit/mixins/Projector.ts

```diff
@@ -167,11 +167,20 @@ registerSuite({
 		assert.equal(projector.root, root);
 	},
 	'pause'() {
-		const projector = new TestWidget();
+
 		let called = false;
-		projector.on('render:scheduled', () => {
-			called = true;
-		});
+
+		class RenderCapturingWidget extends TestWidget {
+			public scheduleRender() {
+				super.scheduleRender();
+
+				if ((<any> this)._scheduled) {
```

Comment:
Could you spy on `global.requestAnimationFrame()` being called rather than accessing a private?